modulemd/v2/include/modulemd-2.0/modulemd-module-stream-v2.h

```diff
@@ -650,4 +651,34 @@ modulemd_module_stream_v2_get_servicelevel (ModulemdModuleStreamV2 *self,
                                             const gchar *servicelevel_name);
 
 
+/**
+ * modulemd_module_stream_v2_add_dependencies:
+ * @self: (in): This #ModulemdModuleStreamV2 object.
+ * @deps: (in): A #ModulemdDependencies object to add to the list for this
+ * module stream.
+ *
+ * Add a #ModulemdDependencies object to the list of dependencies for this
+ * module stream.
+ *
+ * Since: 2.0
+ */
+void
+modulemd_module_stream_v2_add_dependencies (ModulemdModuleStreamV2 *self,
+                                            ModulemdDependencies *deps);
+
+
+/**
+ * modulemd_module_stream_v2_get_dependencies_as_list:
+ * @self: (in): This #ModulemdModuleStreamV2 object.
+ *
+ * Return the list of #ModulemdDependencies objects as a GPtrArray
+ *
+ * Returns: (transfer none) (element-type ModulemdDependencies): A list of
```

Comment:
Yeah, I changed how I was doing this internally and then didn't update the function. This should be renamed to `modulemd_module_stream_v2_get_dependencies()` since it's now returning the internal representation instead of a copied list.